devops/dependencies.json

```diff
@@ -109,4 +109,4 @@
       "root": "{ARCHIVE_ROOT}/comp/oclfpga/win"
     }
   }
-}
+}
```

Comment:
I'm not 100% what is the reason to remove a new line here, but it might be that we need some additional configuration of git.